src/main/java/com/puppycrawl/tools/checkstyle/checks/NewlineAtEndOfFileCheck.java

```diff
@@ -96,6 +109,19 @@
  *   &lt;property name=&quot;lineSeparator&quot; value=&quot;lf&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * // File ending with a new line
+ * public class Test {⤶
+ * ⤶
+ * }⤶ // ok
+ * Note: The comment // ok is a virtual, not actually present in the file
+ *
+ * // File ending without a new line
+ * public class Test1 {⤶
+ * ⤶
+ * }␍⤶ // violation, the file must end with LF only
```

Comment:
The violation message for this case is `Expected line ending for file is LF(\\n), but CRLF(\\r\\n) is detected`, please change description to reflect that here line separator is different, no LF

Reply:
Done